### PR TITLE
fix: syntax error when parsing JSON

### LINKED ISSUE #1009: SyntaxError when parsing JSON
## Error in Trinity Mobile

**SyntaxError** in **[native code]:0**
JSON Parse error: Unrecognized token '<'

[View on Bugsnag](https://app.bugsnag.com/iota-foundation/trinity-mobile/errors/5ad06524d607e6001a8d95f5?event_id=5c53f018003565f377880000&i=gh&m=ci)

## Stacktrace

    [native code] parse
    node_modules/promise/setimmediate/core.js:37:14 tryCallOne
    node_modules/promise/setimmediate/core.js:123:25 
    node_modules/react-native/Libraries/Core/Timers/JSTimers.js:152:6 _callTimer
    node_modules/react-native/Libraries/Core/Timers/JSTimers.js:200:17 _callImmediatesPass
    node_modules/react-native/Libraries/Core/Timers/JSTimers.js:464:11 callImmediates
    [native code] callImmediates
    node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:320:11 spyOrToggle
    node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:135:11 
    node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:297:8 spyOrToggle
    node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:134:17 spyOrToggle
    [native code] value
    [native code] value

[View full stacktrace](https://app.bugsnag.com/iota-foundation/trinity-mobile/errors/5ad06524d607e6001a8d95f5?event_id=5c53f018003565f377880000&i=gh&m=ci)

*Created by Rajiv Shah via Bugsnag*

### PR DESCRIPTION
**Pull request by laumair**
# Description

Should fix https://github.com/iotaledger/trinity-wallet/issues/1009

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

# How Has This Been Tested?

- Manually tested Desktop macOS (debug)

# Checklist:

- [x] My code follows the style guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] For changes to `mobile` that include native code (including React Native modules): I have verified that both iOS and Android successfully build in both `Debug` and `Release` modes
- [ ] For changes to `shared`: If applicable, I have verified that my changes are implemented correctly in `desktop` and `mobile`

### CODE REVIEW

File: src/shared/actions/settings.js [hunk 1]
```diff
@@ -386,6 +377,20 @@ export function getCurrencyData(currency, withAlerts = false) {
                         ),
                     );
                 }
+            })
+            .catch((error) => {
+                console.log('Error', error);
```
Comment (rajivshah3) [inline src/shared/actions/settings.js]:
```suggestion
                // eslint-disable-next-line no-console
                console.log('Error', error);
```

Comment (laumair) [reply src/shared/actions/settings.js]: Ah this wasn't meant to be committed. Removed in https://github.com/iotaledger/trinity-wallet/pull/2013/commits/b4d89a20b334feeb4c6b4dbe3a18d13eefd8bcf7.

File: src/shared/actions/settings.js [hunk 2]
```diff
@@ -386,6 +377,19 @@ export function getCurrencyData(currency, withAlerts = false) {
                         ),
                     );
                 }
+            })
+            .catch((error) => {
```
Comment (rajivshah3) [inline src/shared/actions/settings.js]: `error` is unused (not sure why `eslint` didn't pick this up before)

Comment (laumair) [reply src/shared/actions/settings.js]: Fixed in https://github.com/iotaledger/trinity-wallet/pull/2013/commits/7576877be92b149065a038a3998b6c4df3051c8f

**Review by rajivshah3 — APPROVED** [review]

**Pull request merged**